metaflow/plugins/aws/batch/batch_decorator.py

```diff
@@ -149,6 +149,13 @@ def step_init(self, flow, graph, step, decos, environment, flow_datastore, logge
         if flow_datastore.TYPE != "s3":
             raise BatchException("The *@batch* decorator requires --datastore=s3.")
 
+        # Require iam_role
+        if self.attributes["iam_role"] is None:
+            raise BatchException(
+                "Executions with AWS Batch require an IAM Role.\n"
```

Comment:
Minor nit - `The *@batch* decorator requires @batch(iam_role="role-name") that allows AWS Batch job to communicate with Amazon S3 datastore.`

Reply:
Any reason to remove the mention of configuring this with the environment variable instead? I would suggest
`The *@batch* decorator requires an IAM Role that allows AWS Batch job to communicate with Amazon S3 datastore.\n`
and keeping the other two lines intact

Reply:
You can specify it either with *@batch(iam_role="role-name")* or by setting *METAFLOW_ECS_S3_ACCESS_IAM_ROLE* in your configuration